projects/packages/boost-speed-score/README.md

```diff
@@ -0,0 +1,37 @@
+# boost-speed-score
+
+A package that handles the API to generate the speed score. The `Speed_Score` class registers the API routes that are going to be used in the frontend.
+
+## Get Started
+Build and install the package:
+
+```sh
+jetpack build packages/boost-speed-score && jetpack install packages/boost-speed-score
+```
+
+From the plugin folder, require the package using composer:
+
+```sh
+composer require automattic/jetpack-boost-speed-score
+```
+
+Then use it:
+
+```php
+use Automattic\Jetpack\Boost_Speed_Score\Speed_Score;
+
+$this->speed_scores = new Speed_Score( $modules );
+```
+
+## Using this package in your WordPress plugin
+
+If you plan on using this package in your WordPress plugin, we would recommend that you use[Jetpack Autoloader](https://packagist.org/packages/automattic/jetpack-autoloader) as your autoloader. This will allow for maximum interoperability with other plugins that use this package as well.
```

Comment:
```suggestion
If you plan on using this package in your WordPress plugin, we would recommend that you use [Jetpack Autoloader](https://packagist.org/packages/automattic/jetpack-autoloader) as your autoloader. This will allow for maximum interoperability with other plugins that use this package as well.
```